ramlfications/decorators.py

```diff
@@ -0,0 +1,11 @@
+from .errors import InvalidRAMLError
```

Comment:
How would you feel making this module semiprivate? e.g. `ramlfications/_decorators.py` to signal it's not exactly a public API?